desktop/conf.dist/hue.ini

```diff
@@ -1916,6 +1916,7 @@ submit_to=True
 [[abfs_clusters]]
 # Default ABFS cluster
 [[[default]]]
+## enable_defaultfs_from_coresite=true
```

Comment:
Maybe better to not add this config flag in the .ini files to let the config clean?

Reply:
If a user wants to set the flag and if the flag does not exist in the ini file then the user will not know that there is an option available to set.